meeting-notes/2018/2018-10-29--go-core-dev-team-weekly.md

```diff
@@ -0,0 +1,189 @@
+  - What is the next important thing you should focus on?
+- Ask for general questions. Could be things like:
+  - I'm stuck with something, I don't know who to ask. Who knows who to ask?
+  - Who can help me with xyz?
+- Plan this week
+  - Select issues to work on
+- Review remaining issues if there is time left
+
+
+## Notes
+
+### Week Update
+
+@name
+ - Done:
+ - Blocked:
+ - Next:
+
+
+@marten-seemann
+
+- Done: completed the switch to tls-tris for QUIC (still a bunch of issues to fix though)
+- Blocked: nothing
+- Next: preparing for the IETF meeting next week
+
+@schomatis (not attending)
+- Done:
+  - Solved some issues and reviewed PRs on UnixFS and related components.
+  - Reviewed more UnixFS/MFS code.
+- Next: Coordinate with Rob to start producing some IPFS Files documentation.
+
+@warpfork
+ - Done:
+   - a few hours on refmt obj traversal improvements.
+   - researching prior art for IPLD schemas (protobuf, graphql, fuchsia FIDL) and gathering notes
+     - did you know?  basically nobody allows non-string map keys
+     - did you know?  graphql actually doesn't support maps at all (just structs)
+   - some experimental code for internal reprs of a type system.  just exploring.
+   - reading up on alternative chunkers.  need to digest more.  (ex; adler32 vs buzhash; casync switched from former to latter: why?)
+ - Blocked:
+   - more concrete IPLD data model specs is still on the wishlist.
+ - Next:
+   - need to do some PR reviewing (ex: ipld dagwalker)
+   - figure out where to land some of these notes and design sketches for IPLD
+  
+@travisperson
+ - Done:
+     - first pass at updating iptb
+ - Blocked: N / A
+ - Next:
+     - Mauppdate iptb with a few fixes
+     - update iptb to match ipfs gx deps
+     - release iptb 2.0
+  
+@aschmahmann
+ - Done:
+   - Basic ID-based message passing/protocol structure
+  - Research on GraphSync and planning next steps for graph synchronization related to multi-writer files
+ - Blocked:
+ - Next:
+  - Implement basic graph synchronization
+  - Work towards testing framework
+
+@djdv
+ - Done:
+   - email/response catch up
+   - various FUSE improvements
+   - reviews
+   - reading up on components /subsystems
+ - Blocked:
+   - None this week, some next week
+ - Next:
+   - Handle some outlying PRs (my own)
+   - more reviews
+   - focus on bitswap effort
+   
+@kevina
+  - Done:
+    - First attempt at converting Provider/FindProvider to use multihashes (https://github.com/libp2p/go-libp2p-routing/issues/32)
+  - Blocked:
+    - `--cid-base` option p.r. needs review (https://github.com/ipfs/go-ipfs/pull/5464)
+      right now I just need feedback on the general approach
+    - Need feedback and guidance on converting Provider/FindProvider to use multihashes 
+  - This Week:
+    - Continue work on Provider/FindProvider
+    - Try to get `--cid-base` option p.r. in
+    - Help out with code review
+
+@stebalien
+ - Done:
+    - Fixed race in query events.
```

Comment:
@Stebalien url to PR?

Reply:
https://github.com/libp2p/go-libp2p-routing/pull/31 and in go-ipfs: https://github.com/ipfs/go-ipfs/commit/e41ac96207abfee7eaf3fdb813834cf028d12275

Reply:
Wanna add it to the notes too?

Reply:
Feels a bit overkill but I've added it anyways.